book/06-github/sections/2-contributing.asc

```diff
@@ -260,160 +260,161 @@ Isa sa mga magagandang bagay tungkol sa Git ay maaari mong patuloy na gawin iyon
 
-Let's look at an example. Say we rebased the branch in the previous example, created a new pull request for it, and now we want to reference the old pull request from the new one. We also want to reference an issue in the fork of the repository and an issue in a completely different project. We can fill out the description just like <<_pr_references>>.
+Tingnan natin ang isang halimbawa. Sabihing ating na-rebase ang branch sa nakaraan halimbawa, naglikha ng isang bagong kahilingan na pull para rito, at ngayon gusto natin na ireperensiya ang lumang kahilingan na pull mula sa bago. Gusto din natin na ireperensiya ang isang isyu sa fork ng epositoryo at isyu sa ganap na naiibang proyekto. Maaari nating punan ang paglalarawan kagaya ng <<_pr_references>>.
 
 [[_pr_references]]
-.Cross references in a Pull Request.
+.Mga Pagtukoy sa isang Kahilingan na Pull.
 image::images/mentions-01-syntax.png[PR references]
 
-When we submit this pull request, we'll see all of that rendered like <<_pr_references_render>>.
+Kapag tayo ay nagsumite ng kahilingan na pull na ito, makikita natin ang lahat ng mga naibigay tulad ng <<_pr_references_render>>.
 
 [[_pr_references_render]]
-.Cross references rendered in a Pull Request.
+.Mga pagtukoy na naibigay sa isang Kahilingan na Pull.
 image::images/mentions-02-render.png[PR references rendered]
 
-Notice that the full GitHub URL we put in there was shortened to just the information needed.
+Pansinin na ang buong URL ng GitHub na ating nilagay doon ay pinaikli sa kailangan lamang na impormasyon.
 
-Now if Tony goes back and closes out the original Pull Request, we can see that by mentioning it in the new one, GitHub has automatically created a trackback event in the Pull Request timeline. This means that anyone who visits this Pull Request and sees that it is closed can easily link back to the one that superseded it. The link will look something like <<_pr_closed>>.
+Ngayon kung babalik si Tony at isasara ang orihinal na Kahilingan na Pull, maaari nating makita na sa pagbabanggit nito sa isang bago, awtomatikong nilikha ng GitHub ang isang trackback event sa timeline ng Kahilingan na Pull. Ito ay nangangahilugan na sinuman ang bibisita sa Kahilingan na Pull na ito at makakakita na ito ay naisara ay maaaring madaling maka-link pabalik sa pumalit nito. Magiging tulad ng <<_pr_closed>> ang link.
 
```

Comment:
"Ito ay nangangahilugan na sinuman" change to "Ito ay nangangahulugan na simuman" minimal typo.